net/tlsdial/blockblame/manufacturers.go

```diff
@@ -0,0 +1,41 @@
+package blockblame
```

Comment:
I'd just merge this file in with the other one, really.

Reply:
done